flexmeasures/api/common/schemas/sensor_data.py

```diff
@@ -216,22 +218,37 @@ def load_data_and_make_response(sensor_data_description: dict) -> dict:
         return response
 
     @staticmethod
-    def get_status(sensor: Sensor):
-        end, _ = get_most_recent_clocktime_window(
+    def get_staleness(staleness_search: BeliefsSearchConfigSchema):
```

Comment:
Can you add a type annotation for the return value? For example: `timedelta | None` or `pd.Timedelta | None`.

Reply:
I believe the `staleness_search` should just be a `dict` rather than a `Schema` object.